README.md

```diff
@@ -88,6 +88,34 @@ teams:
     permission: admin
   - name: docs
     permission: push
+
+branches:
+  - name: 'master'
+    # https://developer.github.com/v3/repos/branches/#update-branch-protection
+    # Leaving this empty will disable branch protection for this branch.
+    protection:
```

Comment:
I'm pretty familiar with this page, and some of my recollection of settings has to do with placement:

![screenshot of branch protection config page](https://imgur.com/zdlyFJA.png)

Can we put them in the order of the settings screen as much as possible?

Reply:
We certainly can. That makes a lot of sense. I was ordering it based on the API docs. But basing it on what people are most likely to be familiar with is the right call.